lib/owncloud/test_moveFileStatusCode.py

```diff
@@ -0,0 +1,59 @@
+
+__doc__ = """
+
+Test moving a file via webdav
+
+"""
+
+from smashbox.utilities import *
+
+@add_worker
+def move_non_existing_file(step):
+
+    step(1, 'Create a folder and a file')
+    d = make_workdir()
+    dir_name = os.path.join(d, 'folder')
+    local_dir = make_workdir(dir_name)
+
+    createfile(os.path.join(d, 'file1.txt'), '0', count=1000, bs=50)
+    createfile(os.path.join(local_dir, 'file2.txt'), '1', count=1000, bs=50)
+    run_ocsync(d, user_num=1)
+
+    expect_webdav_exist('file1.txt', user_num=1)
+    expect_webdav_exist(os.path.join('folder', 'file2.txt'), user_num=1)
+
+    step(2, 'Move the file into the folder')
+
+    cmd = 'curl -v -k %s %s -X MOVE -H "Destination:%s" -H "Overwrite:F" ' % (
```

Comment:
any reason why you choose to use "curl" instead of the pyocclient move function ?